src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActions/SuggestedAction.cs

```diff
@@ -112,6 +113,8 @@ private async Task InvokeAsync()
         {
             try
             {
+                using var _ = TelemetryLogging.LogBlockTimeAggregated(FunctionId.SuggestedAction_Application_Summary, "Total");
```

Comment:
how would we know what suggestion action this adds up to?  or are we just wanting to get top level P numbers of invocation in general?